src/runtime/test/globals.spec.tsx

```diff
@@ -7,6 +7,7 @@ describe('globals', () => {
   })
   class CmpA {}
 
+  // eslint-disable-next-line jest/expect-expect
```

Comment:
I believe this is basically implicitly testing that nothing throws an error in here

Reply:
I think so too - can we add justification for future readers here?

```suggestion
  // eslint-disable-next-line jest/expect-expect -- there's not a great way to `expect()` that `raf()` and `setTimeout()` do not throw here
```

Reply:
https://github.com/ionic-team/stencil/pull/3447/commits/32fbf976f4bd5a9298565c1136d47b94ba6eb3dc